insights/parsers/nproc.py

```diff
@@ -46,12 +47,20 @@ def _parse_content(content):
 @parser("limits.conf")
 class LimitsConf(Parser):
     """Parse ``limits.conf`` file contents."""
+    def __init__(cls, *args, **kwargs):
+        deprecated(LimitsConf, "Use the LimitsConf parser in the `limits_conf` module", pending=False)
+        super(LimitsConf, cls).__init__(*args, **kwargs)
+
     def parse_content(self, content):
         self.data = _parse_content(content)
 
 
 @parser("nproc.conf")
 class NprocConf(Parser):
     """Parse ``*-nproc.conf`` file contents."""
+    def __init__(cls, *args, **kwargs):
```

Comment:
Same comment as above regarding use of `cls`.

Reply:
@PaulWay - How do you think about the comments that Bob commented?